packages/crayons-core/src/components/form/form.tsx

```diff
@@ -451,6 +459,18 @@ export class Form {
           : this.values[field] ?? '',
     });
 
+    const fileProps = (field: string, multiple: boolean) => {
+      let value: any = [];
+      if (this.values[field]) {
+        if (multiple) {
+          value = this.values[field];
+        } else {
+          value = this.values[field][0] ? [this.values[field][0]] : [];
+        }
+      }
+      return { value };
```

Comment:
We are returning `[]` even if the type is not multiple and if the `this.values[field]` is not truthy. Is it expected?

Reply:
Yes, this is expected. To keep the type of returning data same. This way anybody implementing it does not have to worry about what the value of multiple prop is.